jax/_src/numpy/ufuncs.py

```diff
@@ -1257,6 +1257,9 @@ def subtract(x: ArrayLike, y: ArrayLike, /) -> Array:
 @implements(np.arctan2, module='numpy')
 @partial(jit, inline=True)
 def arctan2(x1: ArrayLike, x2: ArrayLike, /) -> Array:
+  check_arraylike('arctan2', x1, x2)
+  if dtypes.issubdtype(dtypes.result_type(x1, x2), np.complexfloating):
+    raise TypeError("ufunc 'arctan2/atan2' does not support complex dtypes.")
```

Comment:
Raising an exception does not correspond to deprecation which I would expect to trigger a warning for a few releases. So, I suggest fixing the title of the PR: `Deprecate` -> `Drop` or similar.

Reply:
Thanks! Modified the title.